xAPI-About.md

```diff
@@ -626,15 +629,15 @@ and "category" Context Activities.
      "contextActivities": {
         "category": [
           {
-          "id": "http://w3id.org/xapi/cmi5/context/categories/moveon"
-          },
-          {
-          "id": "http://w3id.org/xapi/cmi5/context/categories/cmi5"
+          "id": "https://w3id.org/xapi/cmi5/context/categories/moveon"
+          },		
+          {		
```

Comment:
Thanks. Will go back and make these edits suggested by Freddie and Andrew.

-------------------------------------------------------
+1.850.266.7100(office)
+1.850.471.1300 (mobile)
jhaag75 (skype)
http://jasonhaag.com (Web)
http://twitter.com/mobilejson (Twitter)
http://linkedin.com/in/jasonhaag (LinkedIn)


On Wed, Apr 20, 2016 at 5:25 AM, Andrew Downes <notifications@github.com>
wrote:

> In xAPI-About.md
> <https://github.com/adlnet/xAPI-Spec/pull/890#discussion_r60384042>:
>
> > @@ -626,15 +629,15 @@ and "category" Context Activities.
> >       "contextActivities": {
> >          "category": [
> >            {
> > -          "id": "http://w3id.org/xapi/cmi5/context/categories/moveon"
> > -          },
> > -          {
> > -          "id": "http://w3id.org/xapi/cmi5/context/categories/cmi5"
> > +          "id": "https://w3id.org/xapi/cmi5/context/categories/moveon"
> > +          },		
> > +          {		
>
> extra tabs added here should be removed
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/adlnet/xAPI-Spec/pull/890/files/f4aa25e37ed3031589d33e303ceef335f0c954be#r60384042>
>